front-end/src/screens/Settings/email.tsx

```diff
@@ -32,15 +34,31 @@ const Email = ({ className }: Props): JSX.Element => {
 	}, [currentUser.email]);
 
 	const onEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => setEmail(event.currentTarget.value);
+	const onPasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => setPassword(event.currentTarget.value);
 
-	const handleClick = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>):void => {
+	const handleEdit = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>):void => {
+		event.preventDefault();
+		setEditing(true);
+	};
+
+	const handleChange = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>):void => {
 		event.preventDefault();
 		event.stopPropagation();
 
+		if (!password) {
+			queueNotification({
```

Comment:
This needs to be displayed in a permanent manner in the UI as an error.

Reply:
done